src/index.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import SearchBox from './components/SearchBox';
+import { Global, css } from '@emotion/core';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const SearchBoxWithStyle = props => (
+  <div>
+    <Global
+      styles={css`
+        * {
+          margin: 0;
+          font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
```

Comment:
Do we really want font styles? This can be inherited from the app user is building.

Reply:
without this, the font is breaking
![image](https://user-images.githubusercontent.com/10362927/66754560-e29ce300-eeb3-11e9-8ad2-462872b57f89.png)

Reply:
Its not breaking its showing the browser default fonts

Reply:
Try using the component on app and check if its getting the inherited fonts or not.